content/terms/navigate-trough-versions-history.en.md

```diff
@@ -1,11 +1,14 @@
 ---
-title: "Browsing through terms"
-weight: 2
+title: "Navigate through versions history"
+weight: 1
+aliases: /navigate-history/
 ---
 
-# Browsing through terms
+# Navigate through versions history
 
-Every collection offers a public database of versions they recorded. 
+{{< video src="https://cloud.opentermsarchive.org/index.php/s/DJ4rqJi5M8fAecL/download/how-to-navigate-through-versions-history-in-open-terms-archive.mp4" >}}
```

Comment:
Add some section title for the video and for the text, for example:


```
## Screencast (4 min)

Learn how to navigate through versions history in this 4 minutes video, or scroll below for the textual version.

<video>

## Manual
```

If this is implemented, then also make sure to indent the `## Notes` below by one header level.